lib/timeline.js

```diff
@@ -176,6 +236,9 @@ Timeline.prototype._request = function(opts, callback) {
   reqOpts.url = this._apiRoot + opts.endpoint;
 
   switch (reqOpts.method) {
+    case 'GET':
+      reqOpts.json = true;
```

Comment:
Since there's no other `GET` API currently, I don't think it needs to be major version change, does it?